theme/static/style/style.css

```diff
@@ -440,6 +487,39 @@ pre code {
         padding-left: 1.8rem;
         text-align: left;
     }
+
+    /*features page*/
+    .features .w50 {
+        padding-top: 2rem;
+    }
+    .features .col-8 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .col-3 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features img {
+        min-width: 95%;
+        max-width: 95%;
+        float: none;
+        margin: auto;
+        display: block;
+    }
+    .features li {
+        padding-left: 1.8rem;
+        padding-right: 1.8rem;
+    }
+    .features .w25 {
+        min-width: 100%;
+        max-width: 100%;
+    }
+    .features .testplan {
+        min-width: 100%;
+        max-width: 100%;
+        margin-bottom: 5rem;
```

Comment:
I think margin-bottom here is redundant, isn't it ?

Reply:
I believe it isn't redundant since we removed the margin-bottom from the now non-existent testplan-img class.